electron/main.ts

```diff
@@ -857,40 +894,44 @@ ipcMain.handle('install', async (event, params) => {
       notify({ title, body: i18next.t('notify.install.canceled') })
       mainWindow.webContents.send('setGameStatus', {
         appName,
+        runner,
         status: 'done'
       })
       return res
     })
 })
 
 ipcMain.handle('uninstall', async (event, args) => {
-  const title = (await Game.get(args[0]).getGameInfo()).title
-  const winePrefix = (await Game.get(args[0]).getSettings()).winePrefix
+  const title = (await Game.get(args[0], args[2]).getGameInfo()).title
+  const winePrefix = (await Game.get(args[0], args[2]).getSettings()).winePrefix
 
-  return Game.get(args[0])
+  return Game.get(args[0], args[2])
```

Comment:
What is args 0 and 2?
Perhaps can we destructure this to a named variable to understand whats it going on here in case someone needs to maintain this code in the future:
```
const something1 = args[0]
const something2 = args[2]
```